docs/source/_static/vendor/new-promotions.css

```diff
@@ -12816,134 +12816,6 @@ only screen and (min-device-pixel-ratio:2) {
     background-color: #fff
 }
 
-@font-face {
```

Comment:
Not used font faces and the files were also not present, causing errors in the console.